index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const app = express();
+const cors = require('cors')
+app.use(cors());
+app.use(express.static('public'))
+app.use(express.json())
+app.use(express.urlencoded({extended: true}))
+
+const {getTubes, getRoute} = require('./functions.js')
+const {response} = require("express");
+const mysql = require("promise-mysql");
+
+app.get('/', getTubes())
```

Comment:
I think it would be better to use a route called `/stations` to get all the stations